contracts/libraries/Token.sol

```diff
@@ -0,0 +1,37 @@
+pragma solidity 0.4.24;
+
+import "openzeppelin-solidity/contracts/token/ERC20/ERC20Basic.sol";
+
+library Token {
+
+    function claimTokens(address _token, address _to) internal {
+        require(_to != address(0));
+        if (_token == address(0)) {
```

Comment:
this part is to claim native coins not tokens consider to have it in a separate internal method.